pombola/settings/tests_south_africa.py

```diff
@@ -29,3 +29,13 @@
     'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
     'LOCATION': 'pmg_api_test',
     }
+
+# Ignore tests for the apps that are not installed
+DIRECTORIES_TO_IGNORE_FOR_TESTS = ['pombola/sms']
+for app in ALL_OPTIONAL_APPS:
+    if app not in INSTALLED_APPS:
+        if app.startswith('pombola.'):
+            DIRECTORIES_TO_IGNORE_FOR_TESTS.append(app.replace('.', '/'))
+
+for directory in DIRECTORIES_TO_IGNORE_FOR_TESTS:
+    NOSE_ARGS += ['--exclude-dir', directory]
```

Comment:
Ignore these directories, otherwise we can exceptions like:

```
ERROR: Failure: RuntimeError (Model class pombola.sms.models.Message doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/nose/loader.py", line 414, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/local/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/local/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/app/pombola/sms/models.py", line 4, in <module>
    class Message(models.Model):
  File "/usr/local/lib/python2.7/site-packages/django/db/models/base.py", line 118, in __new__
    "INSTALLED_APPS." % (module, name)
RuntimeError: Model class pombola.sms.models.Message doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.
```

Reply:
Alternatively, we could put this in a [configuration file](https://github.com/kgrandis/nose-exclude#nose-configuration-files).